Tuist/ProjectDescriptionHelpers/Module.swift

```diff
@@ -857,7 +857,7 @@ public enum Module: String, CaseIterable {
             return .settings(
                 base: [
                     "LD_RUNPATH_SEARCH_PATHS": "$(FRAMEWORK_SEARCH_PATHS)",
-                    "ARCHS": "arm64"
+                    "ARCHS": "arm64",
```

Comment:
shouldn't we include also `x86_64`? fwiw, I'd probably get completely rid of the explicit `ARCHS` and `--architectures` and revert to what we had before

Reply:
Yeah... I'm going to revert because this has had many cascading effects.